lrrbot/stream_status.py

```diff
@@ -33,6 +34,12 @@ def start_check_stream(self):
 	async def check_stream(self):
 		log.debug("Checking stream")
 		data = await twitch.get_info(use_fallback=False)
+		if self.old_title != data['title']:
```

Comment:
If the stream is offline then `'title'` is missing from `data`.

Reply:
Looking at get_info_uncached in common/twitch.py, if the stream is offline the call to helix/streams fails, so it instead fetches from helix/channels, which (according to https://dev.twitch.tv/docs/api/reference#get-channel-information ) also includes the "title" field.

Reply:
Woops, I misread what `get_info()` was doing. If the stream is offline then `data` is `None` because the request to  `GET /helix/channels` is disabled (`use_fallback=False`).

Reply:
Right, so that should change to
```suggestion
		if data and self.old_title != data['title']:
```
I believe...